virtualizarr/backend.py

```diff
@@ -27,9 +28,9 @@
     "zarr_v3": ZarrV3VirtualBackend,
     "dmrpp": DMRPPVirtualBackend,
     # all the below call one of the kerchunk backends internally (https://fsspec.github.io/kerchunk/reference.html#file-format-backends)
-    "netcdf3": NetCDF3VirtualBackend,
     "hdf5": HDF5VirtualBackend,
     "netcdf4": HDF5VirtualBackend,  # note this is the same as for hdf5
+    "netcdf3": NetCDF3VirtualBackend,
```

Comment:
Shouldn't your reader be added here too? Sounds like it could go under `hdf` rather than `hdf5`? Or maybe that's too subtle...

Reply:
Since we discussed treating this new reader as experimental while we do some further battle testing, I'm forcing users to explicitly pass the new backend class `HDFVirtualBackend` to the optional `backend` argument for `open_virtual_dataset` to distinguish it from using the kerchunk reader with filetype auto-detection.